product_docs/docs/biganimal/release/overview/updates.mdx

```diff
@@ -6,7 +6,7 @@ EDB performs periodic maintenance to ensure stability and security of your clust
 
 ## Notification of upcoming maintenance
 
-You're notified in the BigAnimal portal before maintenance occurs. Details are available on the [BigAnimal status page](https://status.biganimal.com/). You can subscribe to get these updates in a feed by selecting **Subscribe to Updates** on the status page. 
+You're notified in the BigAnimal portal before maintenance occurs. Details are available on the [BigAnimal status page](https://status.biganimal.com/). You can subscribe to get these updates in a feed by selecting **Subscribe to Updates** on the status page. You can also enable the notifications to receive in-app or email notifications for upcoming, successful, or failed cluster maintenance upgrade. For more information, see [managing notifications](../getting_started/notifications/#managing-notifications).
```

Comment:
@neel5481 can you help to confirm the statement here is accurate? I am not 100% sure about the "cluster maintenance window" and the entire BigANimal service's maintenance.

Reply:
@xugy99 This looks right to me. As we have 3 types of maintenance notifications

- We notify the customer before 24 hours of maintenance window start which is here upcoming notification
- Once the cluster is upgraded successfully which is successful notification after upgrade
- Cluster upgrade failed which is fail notification

cc: @nidhibhammar